### ISSUE TITLE
Modifying text using Replace Command

### ISSUE THREAD

**Issue by KLauritzen** [notepad-plus-plus/notepad-plus-plus#305]
I am attempting to modify large sections of html files (200 lines+) in folders of hundreds and folders of thousands. Using Replace Command does the modification in these large directories works well in small requests of textual modifications, but in order to modify these larger blocks of text I am having to break up each function in parts because it only fits so much in the line box.

I have also noticed I have to press Replace multiple times (because of the size?) at times to get the function to deliver. The end result is not predictable.

Is there another way I can do this with more efficiency? 

Is there a command that I can bracket in large amounts of text without attempting to fit it in the dialogue box?

**Comment by LouisDotCom** [comment]
We discovered that, doing a simple replace (such as pipes for tabs) in large text files, that the program moved some data around (or changes some line endings)--with the result that we used Visual Studio instead.
The file had 120,020 lines and one corruption occurred at line 12,958. The changes were NOT massive, but enough to prevent the file from being imported into a database. The files are proprietary, so I don't think I can submit a sample, but I did want to note this in case it's helpful to anyone else. Sorry not to be more helpful. (And I really appreciate Notepad++)
